src/validations/css/prettycss.js

```diff
@@ -5,9 +5,34 @@ import endsWith from 'lodash/endsWith';
 
 const RADIAL_GRADIENT_EXPR =
   /^(?:(?:-(?:ms|moz|o|webkit)-)?radial-gradient|-webkit-gradient)/;
+
+const FILTER_VALUE_EXPR =
+  new RegExp([
+    'blur\\(',
+    'brightness\\(',
+    'contrast\\(',
+    'drop-shadow\\(',
+    'grayscale\\(',
+    'hue-rotate\\(',
+    'invert\\(',
+    'opacity\\(',
+    'saturate\\(',
+    'sepia\\(',
+    'inherit',
+    'initial',
+    'unset'].join('|'));
```

Comment:
Anchor makes sense. I was first concerned that this would break stacking filter values ie:
`filter: contrast(175%) brightness(3%);`, but I was wrong.

`initial` and `unset` is noted in the MDN filter documentation: https://developer.mozilla.org/en-US/docs/Web/CSS/filter#Syntax. I've never really used them either so I'm okay with dropping them.